content/_code-samples/tx-serialization/js/helpers.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Since these helpers are only used in one file, I'd recommend just including them in the `index.js` file.